py3-sphinxcontrib-jsmath.yaml

```diff
@@ -32,6 +34,18 @@ pipeline:
       expected-commit: e4c69dd2180c7f18330d5c3fb9300ea0e8461911
       tag: ${{package.version}}
 
+  - name: Disable dev version tagging that causes pip-check failures in rdeps
+    runs: |
+      # These greps will fail when this workaround is no longer needed,
+      # expected to be the next upstream release.
+      grep tag_build setup.py
```

Comment:
the grep fails today. I was unable to build the package with this PR unless I removed both grep calls.